### DISCUSSION TITLE
Welcome to deepmine Discussions!

### DISCUSSION THREAD

**Discussion by daniellovell** [daniellovell/deepmine#1] [General]
## 👋 Welcome!

Here's a thread for any discussions related to the mod's development.
 - feature requests
 - bugs
 - balancing suggestions
 - anything else really

Thanks

**Comment by baratgabor** [comment]
Hi, is it possible to completely disable surface ore spawn? For example not to have any ores in the 0-30 range?

**Comment by Krzeszny** [comment]
Does it work on existing saves?

**Comment by ViroMad** [comment]
dev not responding to comments makes me lack confidence in this mod...
